asv_bench/benchmarks/series_methods.py

```diff
@@ -59,21 +74,27 @@ def time_isin_empty(self):
 
 
 class IsInFloat64:
-    def setup(self):
-        self.small = Series([1, 2], dtype=np.float64)
-        self.many_different_values = np.arange(10 ** 6, dtype=np.float64)
-        self.few_different_values = np.zeros(10 ** 7, dtype=np.float64)
-        self.only_nans_values = np.full(10 ** 7, np.nan, dtype=np.float64)
 
-    def time_isin_many_different(self):
+    params = [np.float64, "Float64"]
+    param_names = ["dtype"]
+
+    def setup(self, dtype):
+        N_many = 10 ** 5
+        N_few = 10 ** 6
+        self.small = Series([1, 2], dtype=dtype)
+        self.many_different_values = np.arange(N_many, dtype=np.float64)
+        self.few_different_values = np.zeros(N_few, dtype=np.float64)
+        self.only_nans_values = np.full(N_few, np.nan, dtype=np.float64)
+
+    def time_isin_many_different(self, dtypes):
         # runtime is dominated by creation of the lookup-table
         self.small.isin(self.many_different_values)
 
-    def time_isin_few_different(self):
+    def time_isin_few_different(self, dtypes):
         # runtime is dominated by creation of the lookup-table
         self.small.isin(self.few_different_values)
 
-    def time_isin_nan_values(self):
+    def time_isin_nan_values(self, dtypes):
         # runtime is dominated by creation of the lookup-table
         self.small.isin(self.few_different_values)
```

Comment:
Not related to your PR, but noticing this now: I suppose the above should use `self.only_nans_values` ?